js/components/portal-dashboard/all-responses-popup/spotlight-dialog.tsx

```diff
@@ -4,9 +4,9 @@ import SpotlightIcon from "../../../../img/svg-icons/spotlight-icon.svg";
 import css from "../../../../css/portal-dashboard/all-responses-popup/spotlight-dialog.less";
 
 interface IProps {
-  handleCloseDialog: (show: boolean) => void;
+  handleCloseNoStudentSelectedDialog: (show: boolean) => void;
```

Comment:
This is probably better named `onCloseDialog` so it becomes:
`onCloseDialog: (show: boolean) => void;`

Reply:
Echoing Matt's point, this follows a React convention that call back props are named onFoo() while the implementations of those callback functions are named handleFoo(), e.g. `onClick={handleClick}`.

Reply:
With currying in `StudentResponsePopup`, we can write it simply as: `onCloseDialog: () => void;`

Reply:
And then in the render simply:
```
          <div className={css.dismissDialogButton} onClick={this.props.onCloseDialog} data-cy="spotlight-dialog-close-button">
            Got it
          </div>
```

Reply:
The render also has some string template literals that are unneeded and could use some improvements to the line breaks around the divs.